js/state/contentSettings.js

```diff
@@ -172,6 +172,16 @@ const getContentSettingsFromSiteSettings = (appState) => {
       addContentSettings(contentSettings.referer, hostPattern, '*', 'allow')
     }
   }
+
+  // temporarySiteSettings
```

Comment:
but i think this has the correct behavior right now since the `runInsecureContent` setting is reset at the end of page load, which means it's not a blocker for merging